packages/atomic/cypress/utils/analyticsUtils.ts

```diff
@@ -0,0 +1,87 @@
+import {CoveoAnalyticsClient} from '@coveo/headless/node_modules/coveo.analytics';
+import {findLast} from './arrayUtils';
+
+export type SearchEventRequest = Parameters<
+  CoveoAnalyticsClient['sendSearchEvent']
+>[0];
+
+export type ClickEventRequest = Parameters<
+  CoveoAnalyticsClient['sendClickEvent']
+>[0];
+
+export type CustomEventRequest = Parameters<
+  CoveoAnalyticsClient['sendCustomEvent']
+>[0];
```

Comment:
I'm essentially just trying to import these types from `coveo.analytics`. importing from `@coveo/headless/node_modules/coveo.analytics/src/events` wasn't allowed by Cypress.

Reply:
Could you import `coveo.analytics` as a direct dev dependencies in this project then ?